<ui version="4.0" >
 <class>QgsUniqueValueDialogBase</class>
 <widget class="QDialog" name="QgsUniqueValueDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>408</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>8</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="mClassVarLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Classification field</string>
     </property>
     <property name="buddy" >
      <cstring>mClassificationComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="mClassificationComboBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>111</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="mClassifyButton" >
       <property name="text" >
        <string>Classify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mDeletePushButton" >
       <property name="text" >
        <string>Delete class</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QListWidget" name="mClassListWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QStackedWidget" name="mSymbolWidgetStack" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="page" />
     <widget class="QWidget" name="page_2" />
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>mClassificationComboBox</tabstop>
  <tabstop>mClassifyButton</tabstop>
  <tabstop>mDeletePushButton</tabstop>
  <tabstop>mClassListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
